OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -73,6 +74,8 @@ public void WorldLoaded(World world, WorldRenderer wr)
 			foreach (var kv in world.Map.ActorDefinitions)
 				Add(kv.Key, new ActorReference(kv.Value.Value, kv.Value.ToDictionary()), true);
 
+			spawnActor = world.WorldActor.Trait<MPStartLocationsInfo>().SpawnPointActor;
```

Comment:
I don't think we can guarantee that `MPStartLocationsInfo` exists here? What happens if we try to open a mission in the editor?

Reply:
Yes, that will crash. I forgot about the editor world. Updated.